cocos/ui/button.ts

```diff
@@ -1057,12 +1068,14 @@ export class Button extends Component {
     }
 
     private _xrClick (): void {
+        if (!USE_XR) return;
```

Comment:
Why not do it like this
```typescript
if (USE_XR) {
    ...
}
```

Than all codes will be removed if USE_XR is false.

Reply:
Their treeshaking results are the same.
`if (USE_XR) {}` will make the original code be intented by 4 space right and it will be more difficult to maintain.
Just adding one line code to treeshake the function is better since we avoid `{`, `}`, `4 spaces indentation`.

Reply:
> Their treeshaking results are the same.

If so, then there is not problem.